source/testnet/guides/become-baker.rst

```diff
@@ -135,6 +110,19 @@ concept of an *epoch* as a unit of time that defines a period in which the set
 of current bakers and stakes are fixed. Epochs have a time duration fixed at the
 Genesis block. In the testnet, epochs have a duration of **1 hour**.
 
+When performing changes, once the transaction that performs the change is
+finalized, this change can be in two different states:
+
+- When a change is said to be **pending** it means the change is processed
+  and will eventually be executed.
+- When a change is said to be **executed** it means the change is performed and
+  will eventually take effect.
```

Comment:
```suggestion
- An **executed** change has been performed and will eventually take effect.
```

Reply:
But what is the difference between "executed", "performed", and "take effect" here? I think we need to clarify.

Reply:
I think this paragraph is exactly trying to explain what the difference between the three states. I think this is clearer in the original wording than in the suggested change, although it is a bit difficult to understand.

I think it would be easier to understand it on a concrete example of baker removal.

Reply:
Section has been removed so this is no longer applicable. Added some tables to try to explain it better.